packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -60,10 +60,9 @@ export function FocusTarget() {
         ) {
           focusAfterSearch = false;
           if (element) {
-            const el = element;
             // The focus seems to be flaky without deferring, especially on iOS.
             defer().then(() => {
-              el.focus();
```

Comment:
`element` is declared outside this scope, and could change during this `defer`. I recall that the reason I created this variable was because I once encountered an instance of that bug.

Reply:
I guess that would mean the element is occasionally being added late. In this case, perhaps `(element ?? el).focus()` would be best.